var/spack/repos/builtin/packages/intel-oneapi-runtime/package.py

```diff
@@ -46,7 +45,8 @@ class IntelOneapiRuntime(Package):
     depends_on("libc", type="link", when="platform=linux")
 
     def install(self, spec, prefix):
-        libraries = get_elf_libraries(compiler=self.compiler, libraries=self.LIBRARIES)
+        oneapi_pkg = self.spec["intel-oneapi-compilers"].package
```

Comment:
can't this also come from `intel-parallel-studio`?

Reply:
No? Currently:
- `intel-parallel-studio` is completely deprecated, and needs to be removed. I didn't do anything to make it compatible with this PR.
- `intel-oneapi-compilers` provides the OneAPI compilers 
- `intel-oneapi-compilers-classic` provides the classic Intel compilers, and depends on `intel-oneapi-compilers` for the artifacts

Basically, I tried not to modify the modeling of Intel compilers from what is already done on `develop` - to avoid extraneous changes in the PR.

Reply:
I think this falls in the realm of "need a compiler toolchain concept anyway so ok, let's close this here.